js/stdlib/ts/ALGO.ts

```diff
@@ -176,7 +176,7 @@ export interface ProviderEnv {
 const defaultALGO_TOKEN_HEADER = 'X-Algo-API-Token';
 const defaultALGO_INDEXER_TOKEN_HEADER = 'X-Indexer-API-Token';
 
-const reachBackendVersion = 21;
+const reachBackendVersion = 22;
```

Comment:
I see this increment, but not in the compiler?

Reply:
forgot to commit it :crying_cat_face: